src/index.html

```diff
@@ -3,9 +3,26 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Hello Front-End</title>
+  <title>PhotoSlideShow</title>
+
+  <script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
+  <script src="ajax.js"></script>
```

Comment:
You'll want the script tags right before the `</body>` tag. Also, `ajax.js` doesn't exist. You'll want to include the source for `js/slide.js`